### PR TITLE
Create Problem2.java

### PR DESCRIPTION
**Pull request by danthesane**
Sum of even Fibonacci #s below 4M - @jart PTAL

### CODE REVIEW

File: Problem2.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+public class Problem2 {
+
+  public static void main(String[] args) {
+    int total = 0;
+    int prevNum = 0;
+    for (int curNum = 1; curNum < 4000000; curNum += prevNum) {
+      prevNum = curNum - prevNum;
+      if ( curNum % 2 == 0) {
+        total += curNum;
+        System.out.printf("%d,", curNum);
```
Comment (jart) [inline Problem2.java]: Did you intend to leave this in?

File: Problem2.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// Print the sum of all even Fibonacci numbers below 4 million
+
+public class Problem2 {
+
+  public static void main(String[] args) {
+    int total = 0;
+    int prevNum = 0;
+    for (int curNum = 1; curNum < 4000000; curNum += prevNum) {
+      prevNum = curNum - prevNum;
+      if ( curNum % 2 == 0) {
```
Comment (jart) [inline Problem2.java]: Extra space

File: Problem2.java (hunk 1 above)
Comment (danthesane) [inline Problem2.java]: Of course, it's a feature.

File: Problem2.java (hunk 2 above)
Comment (danthesane) [inline Problem2.java]: Done

**Comment by jart** [comment]
LGTM assuming this outputs the correct value.

**Pull request merged**